### DISCUSSION TITLE
Does Authentik support work?

### DISCUSSION THREAD

**Discussion by ghost** [goauthentik/authentik#7523] [Q&A] [answered]
Hello.
I tried to contact Authentik by personal and business emails and also registered at the customer portal and submitted a support request there. I also asked in discord. Unfortunately, I still got no answers from the Authentik team.
I understand, this is an open surce product and the resources are limited.
But I considered going enterprise.
And now I am questioning whether support does really work.
Is it a normal behaviour or is it just me not being lucky with Authentik?
Are an enteprise option and support still in an experimental stage?
Please, let me know your your thoughts.
I need to be sure in order to make a product decision.

**Comment by ser** [comment]
Have you paid anything? Do you really expect VIP service without paying a cent? :)

**Comment by ghost** [comment]
Of course, I am not expecting any VIP service of sorts.
But I send an inquiry directly saying that I am ready to become an enterprise client.
It's not even clear how would you become one.
I also asked about terms and what would enterprise support give me.
But got no answer, unfortunately.

**Comment by ghost** [comment, marked as answer]
To summarize, I want to mention that Authentik support replesentative answered my support request on Monday.
